src/samplers/libcall/config.rs

```diff
@@ -0,0 +1,75 @@
+use serde_derive::Deserialize;
```

Comment:
License text should be included at the top of each file. You can grab it from one of the existing files and just set it to 2021 to reflect the year the initial code was written.

Reply:
Added